doc-processor-gradle-plugin/src/test/kotlin/nl/jolanrensen/docProcessor/TestDocContent.kt

```diff
@@ -0,0 +1,157 @@
+package nl.jolanrensen.docProcessor
+
+import io.kotest.assertions.throwables.shouldThrow
+import io.kotest.matchers.shouldBe
+import org.junit.Test
+
+class TestDocContent {
+
+    @Test
+    fun `Simple doc`() {
+        val kdoc = """
+            /**
+             *       Hello World!
+             *
+             * @see [com.example.plugin.KdocIncludePlugin]
+             */
+        """.trimIndent()
+
+        val expected = "\n      Hello World!\n\n@see [com.example.plugin.KdocIncludePlugin]\n"
+
+        kdoc.getDocContentOrNull() shouldBe expected
+
+        kdoc.getDocContentOrNull()?.toDoc() shouldBe kdoc
+    }
+
+    @Test
+    fun `Simple doc no surrounding newlines 1`() {
+        val kdoc = """
+            /** Hello World!
```

Comment:
First line!

Reply:
yes, that's part of the test... it should keep the same layout if you get the content and convert it back to docs